topomodelx/nn/hypergraph/unigcnii.py

```diff
@@ -29,19 +35,37 @@ class UniGCNII(torch.nn.Module):
         https://arxiv.org/pdf/2105.00956.pdf
     """
 
-    def __init__(self, num_classes=2, in_features=1, num_layers=2, alpha=0.5, beta=0.5):
+    def __init__(
+        self,
+        in_channels,
+        hidden_channels,
+        n_layers=2,
+        alpha=0.5,
+        beta=0.5,
+        input_drop=0.2,
+        layer_drop=0.2,
+    ):
         super().__init__()
         layers = []
-        self.num_features = in_features
-        self.num_classes = num_classes
 
-        for _ in range(num_layers):
+        self.input_drop = torch.nn.Dropout(input_drop)
+        self.layer_drop = torch.nn.Dropout(layer_drop)
+        # Define initial linear layer
```

Comment:
Comments are hard to maintain, because people often forget to modify them when they modify code, and that is not caught by unit tests. If you can replace them with variables that are well-named: do it.

for example, here `self.initial_linear_layer` and `self.convolutional_layers` do the job.